apps/mobile/src/screens/GroupWelcomeFlow/GroupWelcomeTabBar.js

```diff
@@ -52,10 +52,7 @@ export default function GroupWelcomeTabBar ({
     }
     const getOutTitle = enforceAgreements ? t('Exit this Group & Return Home') : t('Skip')
     const getOutFunc = enforceAgreements
-      ? () => {
-          navigation.navigate('Group Navigation', { groupSlug: MY_CONTEXT_SLUG })
-          navigation.navigate('Stream', { initial: false })
-        }
+      ? () => openURL(`/groups/${MY_CONTEXT_SLUG}`, { replace: true })
```

Comment:
interesting... so we set it like this so it works for openURL?

On web it would just be `/my`

Reply:
No, it is just wrong. Will correct.